common/tools/cointool.py

```diff
@@ -687,37 +702,47 @@ def check(backend, icons, show_duplicates):
         print("Everything is OK.")
 
 
+type_choice = click.Choice(["bitcoin", "eth", "erc20", "nem", "misc"])
+device_choice = click.Choice(["connect", "suite", "trezor1", "trezor2"])
+
+
 @cli.command()
 # fmt: off
 @click.option("-o", "--outfile", type=click.File(mode="w"), default="-")
 @click.option("-s/-S", "--support/--no-support", default=True, help="Include support data for each coin")
+@click.option("-w/-W", "--wallet/--no-wallet", default=True, help="Include wallet data for each coin")
 @click.option("-p", "--pretty", is_flag=True, help="Generate nicely formatted JSON")
 @click.option("-l", "--list", "flat_list", is_flag=True, help="Output a flat list of coins")
-@click.option("-i", "--include", metavar="FIELD", multiple=True, help="Include only these fields")
-@click.option("-e", "--exclude", metavar="FIELD", multiple=True, help="Exclude these fields")
-@click.option("-I", "--include-type", metavar="TYPE", multiple=True, help="Include only these categories")
-@click.option("-E", "--exclude-type", metavar="TYPE", multiple=True, help="Exclude these categories")
-@click.option("-f", "--filter", metavar="FIELD=FILTER", multiple=True, help="Include only coins that match a filter")
-@click.option("-F", "--filter-exclude", metavar="FIELD=FILTER", multiple=True, help="Exclude coins that match a filter")
+@click.option("-i", "--include", metavar="FIELD (-i shortcut -i name)", multiple=True, help="Include only these fields")
```

Comment:
this is not how metavars work, please move the examples to `help`

Reply:
Sure, moved to `help` in [deb03e6](https://github.com/trezor/trezor-firmware/pull/2286/commits/deb03e6ce0afb5ec3072a572fd74506a4a6ebd9b).

I however quite liked the visuals of `--help` with those examples closer to the definition (and it was more visible in the code as well):
```
  -i, --include FIELD (-i shortcut -i name)
                                  Include only these fields
  -e, --exclude FIELD (-e blockchain_link)
                                  Exclude these fields
```
compared to
```
  -i, --include FIELD             Include only these fields (-i shortcut -i
                                  name)
  -e, --exclude FIELD             Exclude these fields (-e blockchain_link)
```

Reply:
possibly, but the default Click output is a well established Unix CLI convention. doing it differently could, in theory, break some automated help parsers or something like that

for readability, it would help if `help` accepted newlines, but it seems that it doesn't :(